iosxr_iso2vbox.py

```diff
@@ -142,68 +402,33 @@ def main(argv):
     # Handle create OVA
     create_ova = args.create_ova
 
-    # Handle verbose - no special handling needed as boolean
-    verbose = args.verbose
-
     if not os.path.exists(input_iso):
-        print('==>', input_iso, 'does not exist')
-        sys.exit()
+        sys.exit('==>', input_iso, 'does not exist')
 
     # Set Virtualbox VM name from the input ISO
     vmname = os.path.basename(os.path.splitext(input_iso)[0])
 
-    # If verbose is set then print
-    if verbose:
-        def verboseprint(*args):
-            '''
-            If user runs with -v or -verbose print logs
-
-            Print each argument separately so caller doesn't need to
-            stuff everything to be printed into a single string
-            '''
-            print('==> %s: ' % vmname, end="")
-            for arg in args:
-                print(arg,)
-    else:
-        def verboseprint(*args):
-            pass
+    set_logging()
+    logger.setLevel(level=args.verbose)
 
-    def cleanup_vms(name, box_name):
-        '''
-        Cleans up any running box with the passed in name.
-        Unregisters and deletes any box with the passed in box_name
-        '''
-        # Remove any running boxes with the same name
-        vms_list_running = str(run('vboxmanage list runningvms'))
-        if name in vms_list_running:
-            verboseprint("'%s' is running, powering off..." % name)
-            run('VBoxManage controlvm %s poweroff' % name)
-        else:
-            verboseprint("'%s' is not running, nothing to poweroff" % name)
-
-        # Unregister and delete any boxes with the same name
-        vms_list = str(run('vboxmanage list vms'))
```

Comment:
seems there are several more instances with that require changes, please revert all